CostumersAPI/CustomersAPI.csproj

```
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="FluentValidation.AspNetCore" Version="11.2.1" />
+		<PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0" />
+	</ItemGroup>
+
+	<ItemGroup>
+	  <ProjectReference Include="..\AppServices\AppServices.csproj" />
+	  <ProjectReference Include="..\DomainModels\DomainModels.csproj" />
+	  <ProjectReference Include="..\DomainServices\DomainServices.csproj" />
```

Apenas AppService porque AppService possui referencias aos demais
```suggestion
	  <ProjectReference Include="..\AppServices\AppServices.csproj" />
```